Tests/FluentAssertions.Shared.Specs/CollectionEquivalencySpecs.cs

```diff
@@ -17,6 +17,128 @@ namespace FluentAssertions.Specs
     [TestClass]
     public class CollectionEquivalencySpecs
     {
+        #region General
```

Comment:
Probably better to rename this region to "Single Object" or something like that.